scrnaseq.Rmd

```diff
@@ -232,6 +236,10 @@ param$deg_contrasts = data.frame(condition_column=c("orig.ident", "orig.ident",
                                  downsample_cells_n=c(NA, 50, 30),
                                  log2FC=log2(c(1.5, 1.5, 1.5)))
 
+# Enrichr site
+# Available options: "Enrichr", "FlyEnrichr", "WormEnrichr", "YeastEnrichr", "FishEnrichr"
+param$enrichr_site = "Enrichr"
```

Comment:
Can you add this to the `check_enrichr` function so that we always check the correct server?

Reply:
Will do